yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/identity/IdentityException.java

```diff
@@ -0,0 +1,21 @@
+package com.yoti.api.client.spi.remote.call.identity;
+
+public class IdentityException extends Exception {
```

Comment:
What's the benefit of declaring this as a checked exception? We will be forcing the client to wrap the call in a try/catch, but the exception doesn't provide anything beyond what you'd get from a standard runtime exception.

As long as we document that we throw `IdentityException` the client is then free to catch it or let it bubble up. But if we make it checked we _force_ it to always catch it.